src/report.tex

```diff
@@ -211,28 +211,60 @@ \section*{Executive Summary}
 10$^7$ seconds the dose rate in all the port interspaces was greater than
 than 100 $\mu$Sv $hr^{-1}$.  Even though it does not reduce the \gls{sdr} to the
 desired levels, the inclusion of the B$_4$C liner may be an important element
-to a more extensive program of dose reduction in the interspaces.
+to a more extensive program of dose reduction in the interspaces. 
 \\
 \\
 This analysis focused on only shutdown photon sources, and therefore does not
-contain prompt photon doses which are not present following machine shutdown.
+contain prompt photon doses which are not present following machine shutdown. It
+does not contain neutrons that are the product of decay reactions, such as neutron
+production from activated water, which is a small component compared to the plasma
+source. The uncertainty in the \gls{sdr} was estimated being between a relative 
+error of between 0.5 and 2.0.
```

Comment:
this mixes uncertainty and relative error....  How about:
"Although it is not possible to rigorously asses the uncertainty in a calculation of this scope, the uncertainty has been estimated as a factor of 2.  That is, the actual results are expected to be between 0.5 times and 2 times the simulated results."